cmd/feedback/feedback.go

```diff
@@ -80,17 +82,49 @@ type SurveyConfigInterface interface {
 // SurveyStore stores all available surveys.
 // New surveys should be added here.
 var SurveyStore = map[string]SlackSurvey{
-	// PlatformSurvey asks for general developer experience feedback
-	PlatformSurvey: {
-		Name:              PlatformSurvey,
-		PromptDisplayText: "Help make the Slack platform better",
-		PromptDescription: "Tell us about your experience as a Slack developer",
+	// SlackCLIFeedback asks for Slack CLI feedback using GitHub Issues
+	SlackCLIFeedback: {
+		Name:              SlackCLIFeedback,
+		PromptDisplayText: "Slack CLI",
+		PromptDescription: "Questions, issues, and feature requests about the Slack CLI",
+		SkipQueryParams:   true,
+		URL: url.URL{
+			RawPath: "https://github.com/slackapi/slack-cli/issues",
+		},
+		Info: func(ctx context.Context, clients *shared.ClientFactory) {
+			clients.IO.PrintInfo(ctx, false, fmt.Sprintf(
+				"%s\n%s\n",
+				style.Secondary("Ask questions, submit issues, or suggest features for the SLack CLI:"),
+				style.Secondary(style.Highlight("https://github.com/slackapi/slack-cli/issues")),
+			))
+		},
+		Trace: slacktrace.FeedbackMessage,
+		Ask: func(ctx context.Context, clients *shared.ClientFactory) (bool, error) {
+			clients.IO.PrintInfo(ctx, false, style.Sectionf(style.TextSection{
+				Emoji: "love_letter",
+				Text:  "We would love to know how things are going",
+				Secondary: []string{
+					"Share your experience with " + style.Commandf("feedback --name slack-cli-feedback", false),
+				},
+			}))
+			return false, nil
+		},
+		Frequency: Never,
+		Config: func(clients *shared.ClientFactory) SurveyConfigInterface {
+			return clients.Config.SystemConfig
+		},
+	},
+	// SlackPlatformFeedback asks for general developer experience feedback
+	SlackPlatformFeedback: {
+		Name:              SlackPlatformFeedback,
+		PromptDisplayText: "Slack Platform",
+		PromptDescription: "Developer support for the Slack Platform, Slack API, Block Kit, and more",
 		URL:               url.URL{RawPath: "https://docs.slack.dev/developer-support"},
 		Info: func(ctx context.Context, clients *shared.ClientFactory) {
 			clients.IO.PrintInfo(ctx, false, fmt.Sprintf(
 				"%s\n%s\n",
 				style.Secondary("You can send us a message at "+style.Highlight(email)),
-				style.Secondary("Or, survey your experiences at "+style.Highlight("https://docs.slack.dev/developer-support")),
+				style.Secondary("Or, share your experiences at "+style.Highlight("https://docs.slack.dev/developer-support")),
```

Comment:
note: Throughout, I remove the "survey" reference for terms around sharing feedback, since we don't actually link to any surveys now.